utils-module-js/lib/nile.ts

```diff
@@ -0,0 +1,12 @@
+exports.getOrgIDFromOrgName = async function (
+  orgName: String, nile: nileAPI): Promise< string | null > {
+
+  // Check if organization exists
+  var myOrgs = await nile.organizations.listOrganizations()
+  var maybeOrg = myOrgs.find( org => org.name == orgName)
```

Comment:
JS is a crazy language. `==` can cause problems.

The general rule to follow is:

if you want to check the existence of something: `something == null`

to check equality of a `string` or  `number`, you want to use `===`

```suggestion
  var maybeOrg = myOrgs.find( org => org.name === orgName)
```